.gitignore

```diff
@@ -23,3 +23,4 @@ venv/
 pytest-report.xml
 .coverage
 .python-version
+**/local-*
```

Comment:
Consider adding the following:

```
**/charts/*.tgz
```

This will prevent any dependent charts that are utilized in the future from getting checked in.